practice-app/practice/cron.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import requests
+from api.models import Equipment, Parity
+from django_cron import CronJobBase, Schedule
+from django.utils.timezone import make_aware
+from datetime import datetime
+
+
+class GetExchangeRates(CronJobBase):
+    RUN_EVERY_MINS = 5
```

Comment:
what about adding this to settings.py and using like
```suggestion
    from django.conf import settings
    print("Running in every %d mins " % settings.RUN_EVERY_MINS)
```
I guess this is the convenience for django constants

Reply:
I dont think we should do that. RUN_EVERY_MINS parameter is specific to this cronjob, and probably won't change in the future.